src/Tools/dotnet-monitor/Commands/CollectCommandHandler.cs

```diff
@@ -148,6 +148,8 @@ private static IHostBuilder ConfigureServices(this IHostBuilder builder, AuthCon
 
                 services.ConfigureGlobalCounter(context.Configuration);
 
+                services.ConfigureCollectionRuleDefaults(context.Configuration);
```

Comment:
This shouldn't be necessary since ConfigureCollectionRules already calls this.

Reply:
I actually (unintentionally) overloaded `ConfigureCollectionRuleDefaults`; the call here corresponds to a `ConfigureOptions` call, and the one in `ConfigureCollectionRules` is more of a logical grouping for the `IPostConfigureOptions`. I can in-line the latter since it doesn't really serve any practical purpose (and to get rid of the confusion around the names).